docs/vs-2015/snippets/csharp/VS_Snippets_VBCSharp/toolwindowsearch/cs/testtoolwindowsearch_integrationtests/integrationtest library/dialogboxpurger.cs

```diff
@@ -178,7 +178,7 @@ internal bool WaitForDialogThreadToTerminate(int numberOfMillisecondsToWait)
             {
                 lock (Mutex)
                 {
-                    // Set the exit thread to true. Next time the thread will kill itselfes if it sees 
+                    // Set the exit thread to true. Next time the thread will kill itself
```

Comment:
@ghogen It looks like there was a bad find/replace here at some point. Could you look up the original text of this comment?

Reply:
@gewarren I looked at the history in GitHub and in the CAPS code snippet repo before that, and this has been unchanged since 2015 when the sample was first moved into the CAPS code snippet repo, so it just looks like an old typo rather than a recent copy and paste issue.